src/megachatapi_impl.cpp

```diff
@@ -9148,7 +9148,7 @@ const MegaChatGeolocation *MegaChatContainsMetaPrivate::getGeolocation() const
 
 const MegaChatGiphy* MegaChatContainsMetaPrivate::getGiphy() const
 {
-    return mGiphy;
+    return mGiphy ? mGiphy.get() : nullptr;
```

Comment:
Cannot simply return `mGiphy.get()` in all cases
> Returns a pointer to the managed object or nullptr if no object is owned. 
```suggestion
    return mGiphy.get();
```
https://en.cppreference.com/w/cpp/memory/unique_ptr/get